ateam_kenobi/src/play_selector.cpp

```diff
@@ -21,84 +21,123 @@
 
 #include <iostream>
 #include "play_selector.hpp"
+#include "plays/all_plays.hpp"
 #include "ateam_common/game_controller_listener.hpp"
 
 namespace ateam_kenobi
 {
 
+PlaySelector::PlaySelector()
```

Comment:
Just out of curiosity, is there an easy way long term to get plays automatically added to this list and the CMake? It seems like we get slowed down a lot trying to use plays that aren't getting compiled.

Reply:
I think so. I was giving some thought to that too while writing this. A bit of CMake + macro magic should be doable.